float64[] centroid
float64[4] minor_axis
float64 minor_length
float64[4] major_axis
float64 major_length
float64 ang_rad